docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,177 @@
+---
+approvers:
+- davidopp
+- wojtek-t
+title: Pod Priority and Preemption
+---
+
+[Pods](/docs/user-guide/pods) in Kubernetes 1.8 and later can have priority. Priority
```

Comment:
Actually, wasn't priority itself added in 1.7?

Reply:
I think no -- https://github.com/kubernetes/kubernetes/pull/48377 was merged Jul 19.